CHANGELOG.md

```diff
@@ -1,4 +1,21 @@
-## 1.22.0 (Unreleased)
+## 1.23.0 (Unreleased)
+
+
+## 1.22.0 (December 03, 2020)
+
+ENHANCEMENTS:
```

Comment:
go.mod approach?

Reply:
It's not a functional change, I don't think is should be in the change log.
When you have binary, you won't see that.